appinterface/projection/crossfire/crossfire.go

```diff
@@ -531,6 +540,56 @@ func (projection *Crossfire) updateTxSentCount(
 	return nil
 }
 
+func (projection *Crossfire) computePhase1n2CommitmentRank(
+	crossfireValidatorStatsView *view.CrossfireValidatorsStats,
+	crossfireValidatorView *view.CrossfireValidators,
+) error {
+	participants, err := projection.Client.Participants()
```

Comment:
@cdc-Hitesh may I know why this line fails the unit test but your tx_rank_calculation one is okay?

Reply:
Fixed by ghttp